setup.py

```diff
@@ -35,7 +35,7 @@ def get_version(*file_paths: Sequence[str]) -> str:
 
 extras_requirements = {
     'django': ['Django>=2.2.10,<3'],
-    'djangorestframework': ['djangorestframework>=3.10.3,<3.11'],
+    'djangorestframework': ['djangorestframework>=3.10.3,<3.12'],
```

Comment:
@jtrh  I think this file should be executable

Reply:
I'll re-add the execute permission.